operators/awx-operator/2.7.0/metadata/annotations.yaml

```
@@ -0,0 +1,14 @@
+annotations:
+  com.redhat.openshift.versions: v4.10-v4.13n  # Core bundle annotations.
```

It is probably complaining about the `n` at the end actually, not the fact that v4.10 is EOL.  But none-the-less, can you change this to:


```suggestion
  com.redhat.openshift.versions: v4.11  # Core bundle annotations.
```

That way it will support v4.11 and up, meaning the awx-operator will automatically be added to v4.14, and v4.15 when it comes out, etc.